manticore/ethereum.py

```diff
@@ -342,6 +342,22 @@ def check_bitsize(value, size):
         result.append(ABI.make_function_arguments(*args))
         return reduce(lambda x,y: x+y, result)
 
+    @staticmethod
+    def get_uint(data, offset, size):
+        def simplify(x):
```

Comment:
we shouldn't copypaste this everywhere, let's just modify `arithmetic_simplifier`. while we're at it, rename `arithmetic_simplifier` to `arithmetic_simplify`? currently it sounds like it creates a new Simplifier object or something